packages/react-dom/src/__tests__/ReactDOMServerSelectiveHydration-test.internal.js

```diff
@@ -398,11 +407,7 @@ describe('ReactDOMServerSelectiveHydration', () => {
 
     // This should synchronously hydrate the root App and the second suspense
     // boundary.
-    const preventDefault = jest.fn();
-    target.virtualclick({preventDefault});
-
-    // The event should have been canceled because we called preventDefault.
-    expect(preventDefault).toHaveBeenCalled();
```

Comment:
What is this change about?

Reply:
It was to do with how Flare and its usePress worked. I changed this to an alternative version